backend/module/chaos/serverexperimentation/rtds/envoy/envoy.go

```diff
@@ -0,0 +1,237 @@
+package envoy
```

Comment:
this code belongs in a _test.go or mock file

Reply:
yeah after further review this belongs in some kind of mock or test helper package. this will be really useful for future envoy functionality as well. neat stuff.